todo_app/app.py

```diff
@@ -1,7 +1,8 @@
 #from unicodedata import name
+#from crypt import methods
 from flask import Flask, redirect,request,render_template
-from todo_app.data.session_items import add_item#, get_items
-from todo_app.data.trello_items import get_items 
+#from todo_app.data.session_items import add_item#, get_items
```

Comment:
You should remove commented code before merging pull requests. If you ever need the commented bit, you can get it from the git history

Reply:
Commented code has been removed from app.py